index.js

```diff
@@ -22,11 +42,17 @@ const makeHumanReadable = restaurants => new Promise((resolve, reject) => {
     `)));
 });
 
-fetch(`https://developers.zomato.com/api/v2.1/search?entity_id=delhi&q=${keyword}`, {
+fetch(`https://developers.zomato.com/api/v2.1/locations?query=${location}`, {
   method: 'get',
   headers: { 'user-key': key, 'Accept': 'application/json', },
 })
 .then(e => e.json())
+.then(extractLocationData)
+.then(({ entity_id, entity_type }) => fetch(`https://developers.zomato.com/api/v2.1/search?entity_id=${entity_id}&entity_type=${entity_type}&q=${keyword}`, {
+  method: 'get',
+  headers: { 'user-key': key, 'Accept': 'application/json', },
+}))
+.then(e => e.json())
```

Comment:
I think it's a good time to extract these fetch calls into separate functions. Would it be possible for you to do that?

Thanks !

Reply:
I was thinking of doing it. Thought I'd raise a separate PR to clean this up when putting up the PR for `.zomatoclirc`
I'll make the changes ...

Reply:
Let's forget `.zomatoclirc` for now. Although it sounds cool, but keyword@location seems to work well for most use cases. What do you say ?

Reply:
keyword@location looks 🔥 . But I'm thinking of the extensibility of this cli. Are there any other parameters that make sense to be added to the cli?